### PR TITLE
Add RNO_G station description

### PR DESCRIPTION
**Pull request by christophwelling**
Updates detector descriptions for RNO-G

### CODE REVIEW

File: NuRadioReco/detector/RNO_G/RNO_single_station.json [hunk 1]
```diff
@@ -0,0 +1,380 @@
...
+      "ant_comment": "Helper string B",
+      "amp_type": "iglu",
+      "ant_orientation_phi": 0.0,
+      "ant_orientation_theta": 0.0,
+      "ant_position_x": -17.3,
+      "ant_position_y": -10.0,
+      "ant_position_z": -96.0,
+      "ant_rotation_phi": 90.0,
+      "ant_rotation_theta": 90.0,
+      "ant_type": "vpol_4inch_center",
```
Comment (cg-laser) [inline NuRadioReco/detector/RNO_G/RNO_single_station.json:142]: do you have custom local antenna models? shouldn't it be `RNOG_vpol_4inch_center_n1.73`

File: NuRadioReco/detector/RNO_G/RNO_single_station.json [hunk 2]
```diff
@@ -0,0 +1,380 @@
...
+      "ant_comment": "power string Hpol",
+      "amp_type": "iglu",
+      "ant_orientation_phi": 0.0,
+      "ant_orientation_theta": 0.0,
+      "ant_position_x": 0.0,
+      "ant_position_y": 20.0,
+      "ant_position_z": -93.0,
+      "ant_rotation_phi": 90.0,
+      "ant_rotation_theta": 90.0,
+      "ant_type": "RNOG_quadslot_finalMatch",
```
Comment (cg-laser) [inline NuRadioReco/detector/RNO_G/RNO_single_station.json:72]: needs to be renamed to something more meaningful. I suggest `RNOG_quadslot_v3_air_rescaled_to_n1.74`

File: NuRadioReco/detector/RNO_G/RNO_single_station.json (hunk 1 above)
Comment (christophwelling) [reply NuRadioReco/detector/RNO_G/RNO_single_station.json:142]: I think Ilse renamed it when she uploaded it to the server.

**Comment by iplaisier** [comment]
I don't think the RNOG_quadslot_finalMatch is on the server yet (@cg-laser) . The latest is RNOG_quadslot_v2_1.74.

**Comment by cg-laser** [comment]
> I don't think the RNOG_quadslot_finalMatch is on the server yet (@cg-laser) . The latest is RNOG_quadslot_v2_1.74.

do you agree with my proposed name change to RNOG_quadslot_v3_air_rescaled_to_n1.74? Is n=1.74 the index of refraction you used to do the rescaling?

**Comment by iplaisier** [comment]
Yes, i agree and i indeed used 1.74. It might cause some confusion with the previous model (RNOG_quadslot_v2_1.74), but that's an old one anyway.

**Comment by cg-laser** [comment]
ok, antenna pattern is renamed and uploaded. Can you add the hash sum and the documentation of this antenna pattern?

**Comment by iplaisier** [comment]
Yes, I will.

**Comment by cg-laser** [comment]
@christophwelling I saw that the name of the LPDA was just "createLPDA_100MHz". But this should have never worked because NuRadioMC does not append "InfFirn" anymore. Can you confirm?

**Comment by shallmann** [comment]
@cg-laser can't we already merge this or is there something missing? would be super helpful, since people are starting to work towards first data analysis....

**Review by cg-laser — APPROVED** [review]

**Review by cg-laser — CHANGES_REQUESTED** [review]
yes looks good, but the nur file should be removed. I think @christophwelling added it by accident. @shallmann can you remove it? then we can merge.

**Comment by shallmann** [comment]
removed. what about the "createLPDA_100MHz" above?

**Comment by cg-laser** [comment]
I should have fixed all antenna names to the ones available on the server but it would be good if someone could double check.  I'm approving the PR already as I'm pretty sure the current files are correct.

**Review by cg-laser — APPROVED** [review]

**Comment by cg-laser** [comment]
@lpyras is the detector description now correct? Or are there more changed until it can be merged?

**Comment by lpyras** [comment]
> @lpyras is the detector description now correct? Or are there more changed until it can be merged?

Should be done, but the test failed.

**Comment by shallmann** [comment]
@cg-laser I think this is fine now

**Review by lpyras — APPROVED** [review]
Looks good to me.

**Pull request merged**